### DISCUSSION TITLE
“import new scene” by New 3D model

### DISCUSSION THREAD

**Discussion by Shima-Mnz** [Ifsttar/I-Simpa#318] [General]
Dear Sir/Madam
I have problem in  “import new scene” step for New 3D model created. I created the 3D model in AutoCAD 2019 software and exported it in (.stl) format, but when it is imported in the software, I see series of different shapes.

 Please help me as soon as possible.

**Comment by nicolas-f** [comment]
Hi,

You can use also the .3ds format. Check if the right unit is selected (m, cm or mm)

**Comment by Shima-Mnz** [comment]
Hi,
I checked the unit but unfortunately my problem wasn't solved.
You said "You can use also the .3ds format", meaning I use SketchUp software?

**Comment by Pierre-Moulin** [comment]
Hello,

I have created a scene in Solidworks --> One part --> STL 
and it didn't worked.

Must the scene be close ? or could it work with an open one ? 

By the way, I try again with just a close box and it failed again ...

It sayed "Erreur : File '' couldn't be removed (error 3: Le chemin d’accès spécifié est introuvable.)"

Would you help me ?

**Reply by Picaut** [reply]
Yes, the 3D scene must be closed.
It is curious that a close 3D box does not work. Note that the error refers to a problem to access to the 3D file.
Try to export the 3D scene in a PLY file and import it in I-Simpa.
Tell me...

**Reply by Pierre-Moulin** [reply]
It failed again :/ I don't have volume ... maybe a problem in the conversion STL
I will try other things ...
As it is an easy modelling, I am trying to do the structure on I-SIMPA :)
Can't we duplicate and translate a fitting zone ?

**Reply by Pierre-Moulin** [reply]
I have now problem with coincident points. Where could it come from ? 
Must we use Fitting Zone for creating reflecting surfaces ?

![image](https://user-images.githubusercontent.com/106664450/172843681-1e439141-87c5-4d67-9365-8ac9eb54610a.png)

**Reply by nicolas-f** [reply]
Fitting zone describe an area with object sparsly distributed, so it is not a reflecting surface.
You have to add the surfaces into the 3D model

**Reply by Pierre-Moulin** [reply]
My bad, can we create surfaces ? I still having problems with my STL model :(

**Reply by nicolas-f** [reply]
Sorry you can't edit the surfaces in I-Simpa. You have to import a 3D model. Maybe I-Simpa is not compatible with solid works export format.

**Reply by Pierre-Moulin** [reply]
Hello, I finally clear importing my room with PLY :) 
But there is many meshing problems and I have many questions. 
As I have modelling on Solidworks, my room and machines have a thickness of around 2cm 
-What could I do if I want that I SIMPA considers that as a surface ? Just the extern for the machines and the intern for the room.
-How not having problems with coincident points ? 

![image](https://user-images.githubusercontent.com/106664450/173545436-924ff21e-650f-4d92-9743-6e608fd9909f.png)

For you, should I create the scene on I SIMPA and import the machines, or create all and import the room with the machines ?
By the way, I am surprise because I clear importing an open room ? 

![image](https://user-images.githubusercontent.com/106664450/173545729-a1e45421-02f3-4ad3-97b2-b856aedc32fb.png)

Thanks a lot for your support !

**Reply by nicolas-f** [reply]
When you import a scene it will replace the 3d model loaded in I-Simpa. You have to merge your modifications in your 3d model outside of I-Simpa.

**Reply by Pierre-Moulin** [reply]
Hello I clear making my room in surfaces but my machines are always in volumes 
Do you know what could be the cause of this mesh problem ? 
![image](https://user-images.githubusercontent.com/106664450/176688635-d264ffa9-421c-4419-a14a-c3de8b5384af.png)
![image](https://user-images.githubusercontent.com/106664450/176688774-99271c35-627a-437b-a668-69cf995fb0d7.png)

I still having problem with tetramesh .. why ? Must my structure be in the file I-SIMPA ?
C:\Program Files\I-SIMPA\meshing\tetgen / It is here ... 
![image](https://user-images.githubusercontent.com/106664450/176693112-53afd7d9-77de-46b7-a804-443011ad608b.png)

**Comment by nicolas-f** [comment]
You can import/export with compatible software like:

FreeCad (export in ply format)

or

https://app.formit.autodesk.com/

**Comment by Pierre-Moulin** [comment]
Hello 

On Solidworks I have created my machines with surfaces
If I have a problem like this one, is it because I have to select the surface above the machines ? I knew that no as it was part of my floor

![image](https://user-images.githubusercontent.com/106664450/183686349-68f2cf96-c05b-49f5-95af-3453df7ff92a.png)

![image](https://user-images.githubusercontent.com/106664450/183686507-0f469d4e-513b-4afb-a746-3fe8e83499c3.png)

**Reply by Pierre-Moulin** [reply]
and maybe someone could tell me why I have those problems :/ 
There is a link with the mesh ? 

![image](https://user-images.githubusercontent.com/106664450/183697054-87b3b232-5601-4b94-8125-64c740f89f6f.png)

**Reply by nicolas-f** [reply]
Hi,

There is an issue with your 3D model. Duplicate face or self intersection issues.

You can display the location of issues by following the guide:

https://i-simpa-wiki.readthedocs.io/fr/latest/import_file_recommandations.html#use-tetgen-to-identify-problems-in-3d-models

**Reply by Pierre-Moulin** [reply]
See It is fine but do you have an idea of how resolve the problem ? 
Should the machines not touch the floor ? :( 
![image](https://user-images.githubusercontent.com/106664450/183893965-7c9c5d9e-7767-4181-98de-840030235851.png)
